components/application-broker/internal/broker/error.go

```diff
@@ -1,8 +1,17 @@
 package broker
 
+import "github.com/pkg/errors"
+
+type notFoundError struct{}
```

Comment:
Why this error is repated? Check `/internal/storage/driver/memory/error.go`, IMO it should be in one place.

Reply:
I forget to remove it :)